src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -1673,6 +1674,7 @@ sourceFile, getSourceStorageAccountPrimarySharedKey()))
   }
 
   @Test
+  @Disabled
```

Comment:
Make sure to verify that this works; this test is using JUnit 4 annotations and `@Disabled` is JUnit 5. I think the JUnit 4 tests are run using 5 internally so it should work. If it doesn't then using `@Ignored` will work.

Reply:
Oh yeah, I still see it in the test results - https://scans.gradle.com/s/a73cfymlzig3g/tests/overview
<img width="726" alt="image" src="https://github.com/user-attachments/assets/ff6339c2-75b6-4d53-a032-f749c689c421" />

I don't see where we are using Disabled anywhere in TDR.